src/main/java/com/tabnine/binary/requests/notifications/shown/HintShownRequest.kt

```diff
@@ -0,0 +1,26 @@
+package com.tabnine.binary.requests.notifications.shown
+
+import com.google.gson.annotations.SerializedName
+import com.tabnine.binary.BinaryRequest
+import com.tabnine.binary.requests.selection.SetStateBinaryResponse
+import com.tabnine.general.StaticConfig
+
+data class HintShownRequest(
```

Comment:
I'd add a constructor that accepts the `Hover` object and populates its fields to this class's data members